packages/form/src/hooks/use-field.js

```diff
@@ -13,83 +13,49 @@ import { STORE_NAME } from '../data';
 import { useValidation } from './use-validation';
 
 export const useField = ( {
-	name: fieldName,
-	type,
-	value,
-	validation,
+	fieldName,
 	initialValue,
+	isMultiSelect,
+	validation,
 } ) => {
 	const { name: formName } = useContext( Form.Context );
 	const { setFieldValue } = useDispatch( STORE_NAME );
 	const { error, validateField } = useValidation( { fieldName, validation } );
-	const { value: currentValue } = useSelect(
-		( select ) => select( STORE_NAME ).getFieldData( formName, fieldName ),
+	const fieldValue = useSelect(
+		( select ) => {
+			const defaultValue = isMultiSelect ? [] : '';
```

Comment:
If we get rid of `isMultiSelect` you could still pass the default value as a prop when calling the hook.